.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           bundler-cache: true
       - name: Run rubocop
         run: |
-          bundle exec rubocop
+          bundle exec rubocop --ignore-unrecognized-cops
```

Comment:
We don't need to add the CLI option here, as the config should be picked up automatically.